manifests/init.pp

```diff
@@ -55,7 +55,7 @@
   ini_subsetting { 'reports_satellite' :
     ensure               => present,
     path                 => "${::settings::confdir}/puppet.conf",
-    section              => 'master',
+    section              => 'server',
```

Comment:
TIL: the section was renamed. I don't recall seeing this in the changelogs.

Reply:
Have undone this